v2/slogger/retaining_level_filter_appender/retaining_level_filter_appender_test.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2014 MongoDB, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package retaining_level_filter_appender
```

Comment:
[question] really just curious as I'm still a go novice - what's the current standard regarding package structuring? Given we do have provide multiple appenders, my perhaps naive guess would've been to have one common package, say `appenders`, with each of these implementations (with their same filenames) under `/v2/appenders/`? Why package per appender instead?